Resources/input/fsh/ISiKVersicherungsverhaeltnisGesetzlich.fsh

```diff
@@ -25,16 +25,16 @@ Description: "Dieses Profil ermöglicht die Darstellung eines gesetzlichen Versi
     * ^slicing.rules = #open
   * coding contains VersicherungsArtDeBasis 1..1 MS
   * coding[VersicherungsArtDeBasis] = http://fhir.de/CodeSystem/versicherungsart-de-basis#GKV
-* subscriber only Reference(RelatedPerson)
+* subscriber only Reference(RelatedPerson) MS
   * ^definition = "Hauptversicherte Person, wenn abweichend von beneficiary, z.B. bei Familienversicherung"
-  * identifier 1..
+  * identifier 1.. MS // Das MS Flag dient der Übernahme von 'identifier' aus der Patienten-Instanz.
   * identifier only IdentifierKvid10
     * ^short = "VersichertenID (10-stellig) des Hauptversicherten"
     * ^patternIdentifier.system = "http://fhir.de/sid/gkv/kvid-10"
   * display ^short = "Name des Hauptversicherten"
 * beneficiary MS
```

Comment:
wir sagen, dass der beneficiary verpflichten ist, dann sollten wir die Kardinalität auch auf 1.. setzen.

Reply:
beneficiary ist ohnehin 1..1 (siehe https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Basismodul-Stufe-3/markdown-Datenobjekte-Datenobjekte-Versicherungsverhaeltnis?version=current)  ... ob wir das hier setzen is m.M.n eine fsh-stilistische Frage. Dazu habe ich keine Meinung. Faktisch ändern würde es nichts.